Device/Device/schema.json

```diff
@@ -0,0 +1,152 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Device/Device/schema.json",
+  "title": "GSMA / FIWARE - Device schema",
+  "description": "An apparatus (hardware + software + firmware) intended to accomplish a particular task (sensing the environment, actuating, etc.).",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons"
+    },
+    {
+      "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/Location-Commons"
+    },
+    {
+      "$ref": "https://fiware.github.io/dataModels/Device/device-schema.json#/definitions/Device-Commons"
+    },
+    {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "Device"
+          ],
+          "description": "NGSI Entity type"
+        },
+        "controlledAsset": {
+          "type": "array",
+          "items": {
+            "oneOf": [
+              {
+                "type": "string",
+                "format": "uri"
+              },
+              {
+                "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/EntityIdentifierType"
+              }
+            ]
+          }
+        },
+        "mnc": {
+          "type": "string"
+        },
+        "macAddress": {
+          "$ref": "https://fiware.github.io/dataModels/Device/device-schema.json#/definitions/MacAddressType"
+        },
+        "ipAddress": {
+          "type": "string",
+          "oneOf": [
+            {
+              "format": "ipv4"
+            },
+            {
+              "format": "ipv6"
+            }
+          ]
+        },
+        "supportedProtocol": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "enum": [
```

Comment:
we need this enumeration to be part of the commons as well

Reply:
good spot, i missed that, new commit is incoming